README.md

```diff
@@ -51,24 +51,28 @@ To run different web applications for diffent virtual hosts:
 
 
 ## Command-line options
-    Usage: java winstone.Launcher [--option=value] [--option=value] [etc]
-
-    Required options: either --webroot OR --warfile OR --webappsDir OR --hostsDir
-       --webroot                = set document root folder.
-       --warfile                = set location of warfile to extract from.
-       --webappsDir             = set directory for multiple webapps to be deployed from
-    Other options:
+    Usage: java -jar jenkins.war [--option=value] [--option=value]
+    
+    Options:
+       --webroot                = folder where the WAR file is expanded into. Default is ${JENKINS_HOME}/war
+       --pluginroot             = folder where the plugin archives are expanded into. Default is ${JENKINS_HOME}/plugins
+                                  (NOTE: this option does not change the directory where the plugin archives are stored)
+       --extractedFilesFolder   = folder where extracted files are to be located. Default is the temp folder
+       --daemon                 = fork into background and run as daemon (Unix only)
+       --logfile                = redirect log messages to this file
+       --enable-future-java     = allows running with new Java versions which are not fully supported (class version 52 and above)
```

Comment:
When I look at the other options, it seems the "implicit convention" was to use camelCase instead of kebab-case. Not sure it's really related to this PR as I do not see the change of that option.

Reply:
I have no idea why it has to be in Winstone at all. Should be moved to https://github.com/jenkinsci/extras-executable-war IMHO

Reply:
oops my bad I copied the output of jenkins.war :)

Reply:
fixed